.github/workflows/release.yml

```diff
@@ -44,8 +41,8 @@ jobs:
           app_version: ${{ steps.release_details.outputs.INTEGRATION_VERSION }}
           app_name: ${{ steps.release_details.outputs.INTEGRATION_NAME }}
           repo_name: ${{ github.event.repository.full_name }}
-          schema: "custom"
-          schema_url: "https://raw.githubusercontent.com/${{ github.event.repository.full_name }}/${{ env.TAG }}/scripts/s3-publish-schema.yml"
+          schema: "custom-local"
+          schema_path: ./s3-publish-schema-tmp.yml
```

Comment:
This path is not correct.

In the pre-release, it is in this path because we download the artifact made in another job. `make` should leave the schema at `scripts/pkg/s3-publish-schema-tmp.yml`.

Reply:
Thanks! good catch!

Reply:
Updated: [c22873c](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/c22873c24d74e4d8aa195adcd9d1e4a594a69135)